src/main/java/com/backend/allreva/concert/command/domain/Concert.java

```diff
@@ -20,11 +21,10 @@ public class Concert extends BaseEntity {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    private String concertCode;
+    private Long viewCount;
```

Comment:
기본값 세팅을 해주면 좋을 것 같아요!
```suggestion
    @ColumnDefault("0")
```